mobile_back_apple/dev-utils/Makefile

```diff
@@ -131,8 +131,9 @@ tflite-run-sd:
 	bazel-bin/flutter/cpp/binary/main EXTERNAL stable_diffusion \
 		--mode=PerformanceOnly \
 		--output_dir="${REPO_ROOT_DIR}/output" \
-		--model_file="${REPO_ROOT_DIR}/mobile_back_apple/dev-resources/stable_diffusion/sd-models" \
+		--model_file="${REPO_ROOT_DIR}/mobile_back_apple/dev-resources/stable_diffusion/dynamic-sd-models" \
 		--lib_path="bazel-bin/mobile_back_tflite/cpp/backend_tflite/libtflitebackend.so" \
+		--custom_config="stable_diffusion_num_steps:20,stable_diffusion_seed:633994880" \
```

Comment:
Using the `main.cc` file for development, one can pass it with the flag `custom_config`